liquidator/liquidator.js

```diff
@@ -42,6 +42,29 @@ class Liquidator {
         isValid: x => {
           return toBN(x).lt(toBN(toWei("1"))) && toBN(x).gte(toBN("0"));
         }
+      },
+      liquidationDeadline: {
+        // `liquidationDeadline`: Aborts the liquidation if the transaction is mined this amount of time after the
+        // EMP client's last update time. Denominated in seconds, so 300 = 5 minutes.
+        value: 300,
+        isValid: x => {
+          return x >= 0;
+        }
+      },
+      liquidationMinPrice: {
+        // `liquidationMinPrice`: Aborts the liquidation if the amount of collateral in the position per token
+        // outstanding is below this ratio.
+        value: toWei("0"),
+        isValid: x => {
+          return toBN(x).gte(toBN("0"));
+        }
+      },
+      txnGasLimit: {
+        // `txnGasLimit`: Gas limit to set for sending on-chain transactions.
+        value: 9000000, // Can see recent averages here: https://etherscan.io/chart/gaslimit
+        isValid: x => {
+          return x >= 6000000 && x < 15000000;
+        }
```

Comment:
I think these are off by 3 orders of magnitude. `20000000000` is 20 Gwei (IIUC). I think the values here represent 0.009 Gwei, 0.006 Gwei, and 0.015 Gwei respectively.

Reply:
We could also use the `toWei` function's second argument to form these in a more readable way: https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html#towei.

Reply:
Isn't this value the gas limit not the gas price?

Reply:
oh hahaha, I completely misread. My bad. Please ignore. On a related note, it might be useful to allow people to parameterize the gasPrice for these bots (right now I think we default to 20 gwei), but no need to add now.